src/notebooks/phdi_analytics.py

```diff
@@ -0,0 +1,297 @@
+import csv
+import datetime as dt
+import os
+import random
+import re
+
+import censusgeocode as cg
+from concurrent.futures import ThreadPoolExecutor
+from faker import Faker
+from geopy import Point
+from geopy.distance import geodesic
+import geopandas as gpd
+import git
+import matplotlib.ticker as mtick
+import matplotlib.pyplot as plt
+from mpl_toolkits.axes_grid1 import make_axes_locatable
+import names
```

Comment:
This module is imported and used to generate first and last names, but those names are not used in the analytics. I think we should remove this dependency.